python_modules/dagster-external/dagster_external_tests/test_external_execution.py

```diff
@@ -31,12 +31,19 @@ def temp_script(script_fn: Callable[[], Any]) -> Iterator[str]:
         ("stdio", "stdio"),
         ("stdio", "temp_fifo"),
         ("stdio", "fifo"),
+        ("stdio", "temp_file"),
```

Comment:
There is, (stacking multiple `pytest.mark.parametrize` decorators creates a Cartesian product), but I'm holding off because in the CLI PR we can no longer do a straight cartesian product (some combos are invalid).